elements/lisk-chain/test/unit/process.spec.ts

```diff
@@ -650,7 +650,7 @@ describe('blocks/header', () => {
 				).rejects.toMatchObject([
 					expect.objectContaining({
 						message: expect.stringContaining(
-							'Account does not have enough LSK',
+							'Account does not have enough minimum remaining LSK',
```

Comment:
Maybe show the value for minimum?

Reply:
Updated the error message to include actual and expected